scripting/get5/version.sp

```diff
@@ -1,4 +1,4 @@
 #tryinclude "manual_version.sp"
```

Comment:
What's this? `manual_version`?

Reply:
Me neither, I think it should be okay to remove it, I assume it was for development at some point?

Reply:
It's used to stamp manual version numbers (including the build #) in my Jenkins instance. (ctrl+f for 
 manual_version @ https://ci.splewis.net/job/get5/561/console )

I think it would be reasonable to work toward not relying on the jenkins instance for get5 at all, but I'm not quite sure we're there yet: the link in documentation points to https://github.com/splewis/get5/actions, and I don't think it's very easy to find the "latest dev build" from that.

Reply:
Another aside: the directory structure of the github build actions seems excessively nested: `get5.zip/output/get5/addons/sourcemod/...` - I would think just `get5.zip/addons/sourcemod/...` would be preferable, though I haven't checked into how it's configured.

Reply:
@splewis We're planning on getting all of these things fixed as soon as we can merge in 0.9, including having a new official release + easier linking to dev builds. That's why we added the development branch, so we can have only stable releases and tags on master.

Reply:
> Another aside: the directory structure of the github build actions seems excessively nested: `get5.zip/output/get5/addons/sourcemod/...` - I would think just `get5.zip/addons/sourcemod/...` would be preferable, though I haven't checked into how it's configured.

This has been fixed in this CI update as well, where it's simply two attachments in a tag(or pre-release, per GitHub) `get5.zip` and `get5.tar.gz` which only contained the `README` and `LICENSE` at the root directory, followed by the directories `addons`, `cfg` and I believe the translations and whatnot are nested under `addons/sourcemod` so a user can simply unzip the file in their server instance.

As for the documentation, I was planning on linking to the tags since those will sort by most recent and if you click on a tag will show the attachments, and full releases to go to the release page, if that works?